README.md

```diff
@@ -1,6 +1,53 @@
-# FINDOLOGIC export toolkit
+FINDOLOGIC export toolkit
+=========================
 
 [![Travis](https://img.shields.io/travis/findologic/libflexport.svg)](https://travis-ci.org/findologic/libflexport)
 [![Latest Stable Version](https://poser.pugx.org/findologic/libflexport/v/stable)](https://packagist.org/packages/findologic/libflexport)
 
-**Highly experimental! Don't use it!**
\ No newline at end of file
+Synopsis
+========
+
+This project provides a export library for XML and CSV generation according to the FINDOLOGIC export patterns.
+* XML: <https://docs.findologic.com/doku.php?id=export_patterns:xml>
+* CSV: <https://docs.findologic.com/doku.php?id=export_patterns:csv>
+
+
+Basic usage
+============
+
+```PHP
+require_once './vendor/autoload.php';
+
+use \FINDOLOGIC\Export\XML\Exporter;
+use \FINDOLOGIC\Export\Data\Price;
+
+$exporter = Exporter::create(Exporter::TYPE_XML);
+
+$item = $exporter->createItem('123');
+
+$price = new Price();
+$price->setValue('13.37');
+$item->setPrice($price);
+
+$exporter->serializeItems(array($item), 0, 1);
+```
+
+Installation
```

Comment:
"Installation" is a bit German-inspired, so maybe prefer something like "Setup".

Reply:
Done